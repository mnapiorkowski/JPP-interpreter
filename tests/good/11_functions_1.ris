int add(int a, int b):
    return a + b;

bool triple_alternative(bool a, bool b, bool c):
    return a or b or c;

string concat(string a, string b):
    return a + b;

void print_all(int a, bool b, string c):
    println a;
    println b;
    println c;
    return;

void main():
    println add(4, 2);
    println concat("foo", "bar");
    println triple_alternative(false, false, true);
    print_all(4, true, "foo");
    return;