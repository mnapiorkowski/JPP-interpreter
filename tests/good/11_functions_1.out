6
foobar
true
4
true
foo
